Update unreleased.rst (#8223)
crate/crate: blackbox/docs/appendices/release-notes/unreleased.rst

.. When resetting this file during a release, leave the headers in place, but
.. add a single paragraph to each section with the word "None".

> .. Always cluster the topics and link to the documentation whenever possible.  

Reviewer: ```suggestion
.. Always cluster items into bigger topics. Link to the documentation whenever feasible.  
```
